source/class/cv/plugins/diagram/AbstractDiagram.js

```diff
@@ -367,6 +376,10 @@ qx.Class.define('cv.plugins.diagram.AbstractDiagram', {
       check: "Number",
       init: 300
     },
+    forceNowDatapoint: {
```

Comment:
Is this property configurable somehow? I didn't find anythingso it looks like this property is currently obsolete, or is this a preparation for a later change?

Reply:
At the moment it's only using the default value as that's what most use cases will want to use.
But it could be a config option in future where it is changeable.

As it's default value is fixing (i.e. changing) the current behavior of the diagram display I opted to make it easily modifiable for possible future use.